conf/deployment/ibmcloud/ipi_3az_rhcos_3m_3w.yaml

```diff
@@ -0,0 +1,28 @@
+---
+DEPLOYMENT:
+  openshift_install_timeout: 4800
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'ibm_cloud'
```

Comment:
@clacroix12 , do you feel like having same naming convention for platform key as well. I mean, without '_'  i.e  'ibmcloud' ?

Reply:
`ibm_cloud` is already one of our platform values that we use throughout the framework, this wasn't decided with the addition of IPI. See: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L843